.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Coverage Report
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
```

Comment:
In other GitHub action yaml files, we have java-version configured as a variable that gets used in line number 17 and 20. 
Though what you have works, can you make it consistent with other yaml files? This is just a  personal preference.